pkg/crdutil/crdutil.go

```diff
@@ -170,14 +185,19 @@ func applyCRD(
 		if err != nil {
 			return fmt.Errorf("create CRD %s: %w", crd.Name, err)
```

Comment:
Note: we already add `crd.Name` at the caller. Could we just return:
```suggestion
			return fmt.Errorf("create CRD: %w", err)
```
(and similarly for `Update` below)?